tee-worker/litentry/core/assertion-build/src/a4.rs

```diff
@@ -53,6 +67,15 @@ pub fn build(identities: Vec<Identity>, from_date: String, min_balance: f64) ->
 				| VerifiedCredentialsNetwork::Litmus
 				| VerifiedCredentialsNetwork::Ethereum
 		) {
+			let q_min_balance: f64;
+			if verified_network == VerifiedCredentialsNetwork::Litentry
+				|| verified_network == VerifiedCredentialsNetwork::Litmus
+			{
+				q_min_balance = (min_balance / (10 ^ 12)) as f64;
+			} else {
```

Comment:
Please make the target network be more specific. 

if verified_network == VerifiedCredentialsNetwork::Litentry || verified_network == VerifiedCredentialsNetwork::Litmus
{
} else if verified_network == VerifiedCredentialsNetwork::Ethereum
{
}else 
{
continue or skip
}